### PR TITLE
docs: Documentation for Databricks driver

### PR DESCRIPTION
**Pull request by ovr**
Hello!

Thanks

### CODE REVIEW

File: docs/content/Configuration/Connecting-to-the-Database.md [hunk 1]
```diff
@@ -230,6 +231,23 @@ You can connect to a ClickHouse database when your user's permissions are
 [restricted][link-clickhouse-readonly] to read-only, by setting
 `CUBEJS_DB_CLICKHOUSE_READONLY` to `true`.
 
+### Databricks JDBC
+
+Starting from `v0.26.83` Cube.js provides a driver from Databricks. It's based on top of a JDBC driver from DataBricks, which required
```
Comment (rpaik) [inline docs/content/Configuration/Connecting-to-the-Database.md:236]:
```suggestion
Starting with `v0.26.83` Cube.js provides a driver for Databricks. It's on top of a JDBC driver from DataBricks, which required
```

File: packages/cubejs-jdbc-driver/README.md [hunk 2]
```diff
@@ -9,6 +9,39 @@
...
+# Java installation
+
+## macOS
+
+```sh
+brew install openjdk@8
+sudo ln -sfn /usr/local/opt/openjdk@8/libexec/openjdk.jdk /Library/Java/JavaVirtualMachines/openjdk-8.jdk
+```
+
+If it doesn't work, please run commands from `$ brew info openjdk@8`.
```
Comment (rpaik) [inline packages/cubejs-jdbc-driver/README.md:21]:
```suggestion
If this doesn't work, please run commands from `$ brew info openjdk@8`.
```

**Comment by rpaik** [comment]
Just a few minor edits 😄

File: docs/content/Configuration/Connecting-to-the-Database.md [hunk 3]
```diff
@@ -230,6 +231,23 @@ You can connect to a ClickHouse database when your user's permissions are
 [restricted][link-clickhouse-readonly] to read-only, by setting
 `CUBEJS_DB_CLICKHOUSE_READONLY` to `true`.
 
+### Databricks JDBC
+
+Starting with `v0.26.83` Cube.js provides a driver for Databricks. It's on top of a JDBC driver from DataBricks, which required
```
Comment (hassankhan) [inline docs/content/Configuration/Connecting-to-the-Database.md:236]:
```suggestion
Starting with `v0.26.83` Cube.js provides a driver for Databricks. It's based on the JDBC driver from DataBricks, which requires

```

**Review by hassankhan — APPROVED** [review]
Minor nitpicks

**Pull request merged**